tchannel-core/src/main/java/com/uber/tchannel/channels/Peer.java

```diff
@@ -29,8 +29,8 @@
 
 import com.uber.tchannel.api.errors.TChannelConnectionFailure;
 import com.uber.tchannel.api.errors.TChannelError;
-import com.uber.tchannel.messages.InitMessage;
-import com.uber.tchannel.messages.InitRequest;
+import com.uber.tchannel.frames.InitFrame;
```

Comment:
Not sure why it has to be message. They are actually tchannel Frames and FrameTypes.